pkg/server/types.go

```diff
@@ -22,12 +25,15 @@ type BackupRestoreComponentConfig struct {
 	ServerConfig             *HTTPServerConfig                 `json:"serverConfig,omitempty"`
 	SnapshotterConfig        *brtypes.SnapshotterConfig        `json:"snapshotterConfig,omitempty"`
 	SnapstoreConfig          *brtypes.SnapstoreConfig          `json:"snapstoreConfig,omitempty"`
+	SecondarySnapstoreConfig *brtypes.SnapstoreConfig          `json:"secondarySnapstoreConfig,omitempty"`
 	CompressionConfig        *compressor.CompressionConfig     `json:"compressionConfig,omitempty"`
 	RestorationConfig        *brtypes.RestorationConfig        `json:"restorationConfig,omitempty"`
 	HealthConfig             *brtypes.HealthConfig             `json:"healthConfig,omitempty"`
 	LeaderElectionConfig     *brtypes.Config                   `json:"leaderElectionConfig,omitempty"`
 	ExponentialBackoffConfig *brtypes.ExponentialBackoffConfig `json:"exponentialBackoffConfig,omitempty"`
 	DefragmentationSchedule  string                            `json:"defragmentationSchedule"`
+	BackupSyncEnabled        bool                              `json:"backupSyncEnabled,omitempty"`
+	BackupSyncTimeout        time.Duration                     `json:"backupSyncTimeout,omitempty"`
```

Comment:
Not sure if `BackupSyncTimeout` is semantically correct. How about `BackupSyncPeriod`? Thoughts?

```suggestion
	BackupSyncPeriod        time.Duration                     `json:"backupSyncPeriod,omitempty"`
```

Reply:
what about `SecondayBackupSyncPeriod` ? as it's for secondary backup and `BackupSyncPeriod` can be ambiguous as people might think it's for primary backup. wdyt ?